security/pkg/server/grpc/server.go

```diff
@@ -35,11 +35,12 @@ import (
 
 const certExpirationBuffer = time.Minute
 
-// istioCA contains methods to be supported by a CA.
-type istioCA interface {
+// certificateAuthority contains methods supported by a CA.
+type certificateAuthority interface {
 	Sign(csrPEM []byte, ttl time.Duration) ([]byte, error)
 	SignCAServerCert(csrPEM []byte, ttl time.Duration) ([]byte, error)
-	GetCAKeyCertBundle() util.KeyCertBundle
```

Comment:
Quick question: is the returned KeyCertBundle containing the <ca sign cert, ca priv key, root cert>?

Will we always construct a KeyCertBundle for different CA interface implementation? If so, I think can instead add GetCertChainPem/GetRootCertPem to util.KeyCertBundle, and keep CA interface the same.

Reply:
For now, we only have one implementation for KeyCertBundle interface, and no external dependency, so it can be easily changed to struct later.

Reply:
Makes sense. I think KeyCertBundle can be extended to support that.